psydac/api/discretization.py

```diff
@@ -206,7 +205,6 @@ def discretize_derham_multipatch(derham, domain_h, *args, **kwargs):
             for V, basis in zip(derham.spaces, bases)]
 
     return DiscreteDerhamMultipatch(
-        mapping  = mapping,
```

Comment:
since we are no longer passing `mapping` as an argument, we don't need to define it above I guess